content/docs/guides/usage_reporting.md

```diff
@@ -0,0 +1,36 @@
++++
```

Comment:
This is great, thanks for doing this! I have just one suggestion: Change the file name to use a hyphen instead of an underscore:
usage-reporting.md

Using a hyphen rather than an underscore in URLs is common practice. I see we use hyphens for all the file names in the getting-started section, and it'd be good to standardise.
https://github.com/kubeflow/website/tree/master/content/docs/started

Now that you're removing the old "user_guide.md", this is a good time to start the standardisation.  :)